Boilerplate doc (#290)
vhive-serverless/STeLLAR: README.md

7. Benchmarking is performed.
8. Once all the experiments are finished, the service is removed using the serverless.com framework. (`serverless remove`)

### Deployment boilerplate outline
Below are listed key packages and components that are most essential to serverless.com framework deployment.
 - src/setup/
   - extract-configuration.go

>      - Takes the experiment.json file and assigns according values to Go Configuration struct.

Reviewer: ```suggestion
     - Reads the experiment.json file and assigns values accordingly to Go Configuration struct.
```

   - assign-endpoints.go

>      - Assigns endpoints and routes to each function deployed. Endpoints and routes are also stored in the Configuration struct. The enpoint values are extracted form the serverless framework deploy message

Reviewer: ```suggestion
     - Assigns endpoints and routes to each function deployed. Endpoint values are extracted from the Serverless framework deploy message and stored in the Configuration struct.
```

   - serverless-config.go
     - Creates the serverless.yml file based on the data in the experiment json file.

 - src/setup/building/

>    - This package takes care of building Go binaries and Gradle projects for Java deployment.

Reviewer: ```suggestion
   - This package takes builds Go binaries and Gradle projects for Java deployment.
```

 - src/setup/code-generation/
   - This package generates Hello World provider-specific function source-code used for deployment.
 - src/setup/deployment/packaging/

>    - This package takes care of creating filler files, zipping of the function package and creation of artifacts.

Reviewer: ```suggestion
   - This package creates filler files, zips function packages and creates artifacts.
```

 - src/setup/deployment/connection/

>    - This package contains legacy code from the original deployment implementation, which used provider specific APIs. Most of this package will be eventually obsolete, but we may get use of some of its functionalities.

Reviewer: ```suggestion
   - This package contains legacy code from the original deployment implementation, which used provider specific APIs. Most of this package will be eventually obsolete and deprecated, but we may get use of some of its functionalities.
```
